cli/cli.go

```diff
@@ -935,7 +935,7 @@ func (c *cmd) action(err error, f func(*cli.Context) error, daemonURL string, la
 				os.Exit(1)
 			case internal.ErrSocketAccessDenied:
 				color.Red(formatError(internal.ErrSocketAccessDenied).Error())
-				color.Red("Run 'usermod -aG nordvpn $USER' to fix this issue and log out of OS afterwards for this to take an effect.")
+				color.Red("Run 'usermod -aG nordvpn $USER' to fix this issue and reboot your device afterwards for this to take an effect.")
```

Comment:
Rytis (the Meshnet writer who encountered the original issue) also suggested adding "Run '**sudo** usermod -aG nordvpn $USER'" to the message, as casual users often get confused why the copy-pasted command doesn't work.

Arguments for:
1. Would make it simpler and more clear for casual Ubuntu users (the biggest share of our users I believe)
Arguments against:
1. `sudo` is not always necessary: in Docker or in plain Debian for example. On the other hand users using these are most likely more advanced and can figure that out by themselves.
2. Copy pasting commands with `sudo` that you don't understand is probably not a good practice which we shouldn't encourage. 

I would still vote for including `sudo` in the example command because that just seems more pragmatic to me, but more opinions are welcome.

Reply:
Actually I had it with `sudo` but accidentally reverted that. Also on the [website](https://support.nordvpn.com/Connectivity/Linux/1325531132/Installing-and-using-NordVPN-on-Debian-Ubuntu-Raspberry-Pi-Elementary-OS-and-Linux-Mint.htm) it contains `sudo`.

So I'm also for `sudo` :)